src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.js

```diff
@@ -34,18 +46,38 @@ const AnnotationActivityMenu = ({ canDelete, canEdit, id, onDeleteConfirm, onEdi
         onDeleteConfirm();
     };
 
+    const handleMenuClose = () => {
+        setIsMenuOpen(false);
+    };
+
+    const handleMenuOpen = () => {
+        setIsMenuOpen(true);
+    };
+
     const tetherProps = {
         attachment: 'top right',
         className: 'bcs-AnnotationActivity-deleteConfirmationModal',
         constraints: [{ to: 'scrollParent', attachment: 'together' }],
         targetAttachment: 'bottom right',
     };
 
+    React.useEffect(() => {
+        if (isConfirmingDelete || isMenuOpen) {
```

Comment:
Is the `isConfirmingDelete` check needed? It looks like the menu is still open in that scenario, just disabled.

Reply:
Yeah I need to tweak this effect a bit